src/Geocoder/Provider/GoogleMaps.php

```diff
@@ -153,8 +153,9 @@ private function executeQuery($query)
             throw new NoResult(sprintf('Could not execute query "%s".', $query));
         }
 
-        if ('REQUEST_DENIED' === $json->status && 'The provided API key is invalid.' === $json->error_message) {
-            throw new InvalidCredentials(sprintf('API key is invalid %s', $query));
+        if ('REQUEST_DENIED' === $json->status) {
```

Comment:
I mean, if the message has changed, then let's change the message in the code rather than removing it.